core/templates/domain/review_test/review-test-backend-api.service.ts

```diff
@@ -35,16 +49,21 @@ export class ReviewTestBackendApiService {
     private http: HttpClient
   ) {}
 
-  _fetchReviewTestData(storyId: string): Promise<Object> {
-    return this.http.get(
+  _fetchReviewTestData(storyId: string): Promise<ReviewTestData> {
+    return this.http.get<ReviewTestBackendResponse>(
       this.urlInterpolationService.interpolateUrl(
         ReviewTestDomainConstants.REVIEW_TEST_DATA_URL,
         {story_id: storyId}
       )
-    ).toPromise();
+    ).toPromise().then(backendResponse => {
+      return {
+        storyName: backendResponse.story_name,
```

Comment:
We are not returning domain object here, right?

Reply:
Yeah, you're right we can return a domain object. I'll refactor it.

Reply:
done